encryptedlogging/build.gradle.kts

```diff
@@ -56,16 +61,20 @@ fun loadPropertiesFromFile(file: File): Properties {
 dependencies {
     implementation(libs.androidx.core.ktx)
     implementation(libs.androidx.appcompat)
+    implementation(libs.androidx.room.runtime)
+    ksp(libs.androidx.room.compiler)
+    api(libs.androidx.room.ktx)
```

Comment:
👋 What's the reason for making this `api`? I don't think that "encrypted logging" library should need its client to do anything regarding the databasae - if current design enforces `api` here, let's refactor the solution.

Reply:
No reason @wzieba, this is just a quick copy-paste (mistake) I did while getting this dependencies as is from another project (I think WCAndroid), just to test this asap, noted and thanks! 🔖